Standards/scs-XXXX-vN-standard-template.md

```diff
@@ -34,13 +41,13 @@ Option 1 description
 
 Option 2 description
 
-## Open questions
+### Open questions
 
 RECOMMENDED
 
-## Decision
+## Standard
 
-Decision
+Standard
```

Comment:
```suggestion
What is the essence of this standard? Adjust heading accordingly.
```

Reply:
Changed in 6fb17a19fc38f70a0dda6c5cc9689ff28d0b31e4